src/pages/Lend/LoanChannelCategoryPage.vue

```diff
@@ -160,6 +160,22 @@ export default {
 							});
 						}).then(() => {
 							return fetchExperimentSettings('eco_challenge', client);
+						}),
+						client.query(
+							{
+								query: experimentAssignmentQuery,
+								variables: { id: 'loan_channel_flss_query_v1' }
+							}
+						).then(() => {
+							return client.mutate({
+								mutation: updateExperimentVersion,
+								variables: {
+									id: 'loan_channel_flss_query_v1',
+									version: 'b'
+								}
+							});
+						}).then(() => {
+							return fetchExperimentSettings('loan_channel_flss_query_v1', client);
```

Comment:
Isn't this just doing the same thing you've done above?

Reply:
This is the only sequence I found that works. You can't mutate an unassigned experiment.

So the sequence is assign it, mutate it, then run the assign query again so that we can read this query from the cache where we need to which returns the mutated version.